x/examples/website-wrapper-app/wrapper_app_project/.scripts/build.mjs

```diff
@@ -46,17 +37,32 @@ const DEFAULT_SMART_DIALER_CONFIG = {
 
 export default async function main(
   {
-    platform,
+    additionalDomains = [],
+    appId,
+    appName,
     entryDomain = "www.example.com",
     navigationUrl,
+    output = OUTPUT_DIR,
+    platform,
     smartDialerConfig = DEFAULT_SMART_DIALER_CONFIG,
-    additionalDomains = [],
   },
 ) {
+  // Manually expand the '~' character in the output path.
+  output = output.replace("~", process.env.HOME);
```

Comment:
Expanding `~` correctly is challenging. For example, `~/test` and `~test` can resolve to different directories.

Maybe we should remove this and expect users to perform the expansion themselves in their shell?